UserManagement/accounts/repositories/interfaces.py

```diff
@@ -0,0 +1,51 @@
+from abc import ABC, abstractmethod
+
+
+class IUserRepository(ABC):
```

Comment:
and read about repository patter, you could not explain today what you did for the project 
for example here - https://www.cosmicpython.com/book/chapter_02_repository.html